traffic_ops/app/db/seeds.sql

```diff
@@ -541,6 +541,10 @@ insert into api_capability (http_method, route, capability) values ('GET', 'deli
 insert into api_capability (http_method, route, capability) values ('POST', 'deliveryserviceserver', 'delivery-service-servers-write') ON CONFLICT (http_method, route, capability) DO NOTHING;
 insert into api_capability (http_method, route, capability) values ('POST', 'deliveryservices/*/servers', 'delivery-service-servers-write') ON CONFLICT (http_method, route, capability) DO NOTHING;
 insert into api_capability (http_method, route, capability) values ('DELETE', 'deliveryservice_server/*/*', 'delivery-service-servers-write') ON CONFLICT (http_method, route, capability) DO NOTHING;
+-- delivery service server capabilities
+insert into api_capability (http_method, route, capability) values ('GET', 'deliveryservice_required_capabilities', 'delivery-services-read') ON CONFLICT (http_method, route, capability) DO NOTHING;
```

Comment:
these need to be updated as well

Reply:
Thanks it got left out on accident with the first commit